driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1295,6 +1296,24 @@ public Builder withNettyOptions(NettyOptions nettyOptions) {
             return this;
         }
 
+        /**
```

Comment:
Following the [python driver implementation](https://github.com/datastax/python-driver/pull/846), NO_COMPACT is not enabled by default.